<ui version="4.0" >
 <class>LocationEditor</class>
 <widget class="QDialog" name="LocationEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Location Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Locations</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="0" column="1" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="label_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Information</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QListWidget" name="m_LocationList" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="m_AddButton" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_DeleteButton" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item rowspan="2" row="1" column="2" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_GeneralTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>246</width>
        <height>309</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>General</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <layout class="QGridLayout" name="gridLayout" >
         <property name="sizeConstraint" >
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="verticalSpacing" >
          <number>0</number>
         </property>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="m_IDEdit" >
           <property name="toolTip" >
            <string>ID used within the script.</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_5" >
           <property name="text" >
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="m_NameEdit" >
           <property name="toolTip" >
            <string>String displayed on the Move screen.</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Background ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Internal ID</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="m_BGCB" />
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_HotspotsTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>246</width>
        <height>309</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Hotspots</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QPushButton" name="m_AddSpotButton" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_DeleteSpotButton" >
           <property name="text" >
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="m_HotspotList" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_StatesTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>246</width>
        <height>309</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>States</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <widget class="QPushButton" name="m_AddStateButton" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_DeleteStateButton" >
           <property name="text" >
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="m_StateList" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>line</zorder>
  <zorder>label_2</zorder>
  <zorder>m_LocationList</zorder>
  <zorder>buttonBox</zorder>
  <zorder></zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LocationEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LocationEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
